common-test/src/main/java/net/chrisrichardson/eventstore/examples/customersandorders/commontest/AbstractCustomerAndOrdersIntegrationTest.java

```diff
@@ -54,38 +54,42 @@ public void shouldCreateAndApproveOrder() throws Exception {
           return cv;
         });
 
-        assertEquals(creditLimit, customerView.getCreditLimit());
-        assertEquals(orderTotal, customerView.getOrders().get(orderId).getOrderTotal());
-
+        eventually(120, 500, TimeUnit.MILLISECONDS, () -> {
+          assertEquals(creditLimit, customerView.getCreditLimit());
+          assertEquals(orderTotal, customerView.getOrders().get(orderId).getOrderTotal());
+        });
     }
 
     @Test
-    public void shouldCreateAndRejectOrder() {
+    public void shouldCreateAndRejectOrder() throws Exception {
 
-        String customerId = createCustomer(creditLimit);
+      String customerId = createCustomer(creditLimit);
 
-        Money orderTotal = creditLimit.add(new Money(1));
+      Thread.sleep(4000);
```

Comment:
Why sleep?

Reply:
Chris, there are race conditions: https://github.com/eventuate-examples/eventuate-quarkus-examples-customers-and-orders/issues/6